djstripe/settings.py

```diff
@@ -150,6 +151,23 @@ def get_subscriber_model():
     return subscriber_model
 
 
+def get_api_version():
+    """Get the desired API version to use for Stripe requests."""
+    version = getattr(settings, 'DJSTRIPE_API_VERSION', None) or '2017-02-14'
```

Comment:
Annoyingly this is what it's implemented as in our hacked-up (non dj-stripe) code. ;-)  Makes sense, I'll change it, let's be consistent.